pyani_plus/private_cli.py

```diff
@@ -147,7 +146,6 @@ def log_configuration(  # noqa: PLR0913
         program=program,
         version=version,
         fragsize=fragsize,
-        maxmatch=maxmatch,
```

Comment:
`-mum` should be the default option for ANIm for consistency with previous versions and as suggested by MUMmer docs. We can either drop `-maxmatch` as an option, with `-mum` as the default behaviour, or we can keep it.